thorlcr/thorutil/thormisc.hpp

```diff
@@ -606,6 +608,107 @@ extern graph_decl bool getExpertOptBool(const char *opt, bool dft=false);
 extern graph_decl __int64 getExpertOptInt64(const char *opt, __int64 dft=0);
 extern graph_decl StringBuffer &getExpertOptString(const char *opt, StringBuffer &out);
 
+////
+// IContextLogger
+class CThorContextLogger : implements CSimpleInterfaceOf<IContextLogger>
```

Comment:
trivial: but when we're deriving from an abstract interface, we usually use 'implements', but CSimpleInterfaceOf is a partial implementation of an interface, so we tend to use 'public' (although I do see a couple of exceptions).